apisix/plugins/jwt-auth.lua

```diff
@@ -407,6 +434,27 @@ function _M.rewrite(conf, ctx)
         return 401, {message = "failed to verify jwt"}
     end
 
+    -- check for hiding `Authorization` request header if `hide_credentials` is `true`
+    if conf.hide_credentials then
+        -- hide sensitive field
+        if from_header then
+            -- hide for header
+            local temp_token = core.request.header(ctx, conf.header)
+            core.request.set_header(ctx, conf.header, nil)
+
+
```

Comment:
Redundant empty lines? I think one line is enough.

Reply:
Thx for your advice, I will cover it.